src/temp_file.cpp

```diff
@@ -4,7 +4,7 @@
 #include <boost/filesystem.hpp>
 
 namespace miopen {
-TempFile::TempFile(const std::string& path_template) : name(path_template), dir("tmp")
+TempFile::TempFile(const std::string& path_infix) : name("file"), dir(path_infix)
```

Comment:
This changes directory name from "`miopen-tmp-...`" to "`miopen-<INFIX>-...`" which is more informative for the geeks who would like to inspect the content of the system temp directory.